<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UniteDialog</class>
 <widget class="QDialog" name="UniteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Split Images</string>
  </property>
  <layout class="QVBoxLayout" name="plContent">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <widget class="QWidget" name="pwToolButtons" native="true">
     <layout class="QHBoxLayout" name="plToolButtons" stretch="0,0,0,0">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="prImage1">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pbZoomIn">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../GalleryExplorer.qrc">
          <normaloff>:/res/resources/increase.png</normaloff>:/res/resources/increase.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pbZoomOut">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../GalleryExplorer.qrc">
          <normaloff>:/res/resources/decrease.png</normaloff>:/res/resources/decrease.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="prImage2">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="pwImages" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="plImages">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QScrollArea" name="psImage1">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QLabel" name="plImage1">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>389</width>
           <height>517</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="psImage2">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QLabel" name="plImage2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>389</width>
           <height>517</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="pwButtons" native="true">
     <layout class="QHBoxLayout" name="plButtons">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pbUnite">
        <property name="text">
         <string>Unite</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="pwButtonsFree" native="true"/>
      </item>
      <item>
       <widget class="QPushButton" name="pbCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../GalleryExplorer.qrc"/>
 </resources>
 <connections/>
</ui>
